### PR TITLE
pfsreadwrite file OK, contpfs OK and data/bss migrated, pfsgetstatus OK

### PR DESCRIPTION
**Pull request by kyleburnette**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

I meant to do only one at a time, but these kinda relied on each other

### CODE REVIEW

**Review by AngheloAlf — APPROVED** [review]

File: include/functions.h [hunk 1]
```diff
@@ -305,8 +305,8 @@ void osViBlack(u8 active);
 s32 __osSiRawReadIo(u32 devAddr, u32* data);
 OSId osGetThreadId(OSThread* t);
 void osSpTaskYield(void);
-s32 __osPfsGetNextPage(OSPfs* param_1, __OSInode* param_2, u8 param_3, u8 param_4);
-s32 osPfsReadWriteFile(OSPfs* pfs, s32 file_no, u8 flag, s32 offset, s32 size_in_bytes, u8* data_buffer);
+s32 __osPfsGetNextPage(OSPfs* pfs, u8* bank, __OSInode* inode, __OSInodeUnit* page);
+s32 osPfsReadWriteFile(OSPfs* pfs, s32 fileNo, u8 flag, s32 offset, s32 size, u8* data);
```
Comment (Kenix3) [inline include/functions.h:309]:
```suggestion
s32 osPfsReadWriteFile(OSPfs* pfs, s32 fileNo, u8 flag, s32 offset, size_t size, u8* data);
```

File: src/libultra/io/pfsreadwritefile.c [hunk 2]
```diff
@@ -1,5 +1,124 @@
...
+    if (!CHECK_IPAGE(*page, *pfs)) {
+        if (page->ipage == PFS_EOF) {
+            return PFS_ERR_INVALID;
+        }
+        return PFS_ERR_INCONSISTENT;
+    }
+    return 0;
+}
+
+s32 osPfsReadWriteFile(OSPfs* pfs, s32 fileNo, u8 flag, s32 offset, s32 size, u8* data) {
```
Comment (Kenix3) [inline src/libultra/io/pfsreadwritefile.c:28]:
```suggestion
s32 osPfsReadWriteFile(OSPfs* pfs, s32 fileNo, u8 flag, s32 offset, size_t size, u8* data) {
```

**Review by Kenix3 — CHANGES_REQUESTED** [review]

**Comment by EllipticEllipsis** [comment]
As well as Kenix's comments, this needs some variable names fixing, according to Jenkins.

File: src/libultra/io/pfsreadwritefile.c (hunk 2 above)
Comment (kyleburnette) [reply src/libultra/io/pfsreadwritefile.c:28]: If I make this change, several places don't match anymore. In this function it specifically checks that size is greater than 0, so maybe they didn't actually use u32/size_t?

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**